rm_vk/rm_vk/MyGroupsTableViewController/MyGroupsTableViewController.swift

```diff
@@ -0,0 +1,85 @@
+// MyGroupsTableViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран списка групп пользователя
+final class MyGroupsTableViewController: UITableViewController {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let cellIdentifier = "MainTableViewCell"
+        static let oneGroupName = "homer"
+        static let twoGroupName = "marge"
+        static let oneGroupTitle = "We love Homer"
+        static let twoGroupTitle = "Blue Hair"
+        static let oneGroupDescription = "клуб любителей гомера"
+        static let segueIdentifier = "addGroup"
+    }
+
+    // MARK: - Private Properties
+
+    private var groups = [
+        Group(
+            avatarImageName: Constants.oneGroupName,
+            title: Constants.oneGroupTitle,
+            description: Constants.oneGroupDescription
+        ),
+        Group(
+            avatarImageName: Constants.twoGroupName,
+            title: Constants.twoGroupTitle,
+            description: nil
+        )
+    ]
+
+    // MARK: - Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        registerCell()
+    }
+
+    @IBAction func addGroup(segue: UIStoryboardSegue) {
```

Comment:
```suggestion
  // MARK: - Private IBAction
    @IBAction private func addGroup(segue: UIStoryboardSegue) {
```
ты пробовала этот метод private сделать? и марку поставь